tests/test_aaa_authorization_service.rb

```diff
@@ -22,54 +22,72 @@ class TestAaaAuthorizationService < CiscoTestCase
   def setup
     super
     feature_tacacs
+    preconfig_tacacs_server_access(tacacs_groups[0])
+    config_tacacs_servers(tacacs_groups[1..3])
   end
 
   def teardown
+    cleanup_aaa
     feature_tacacs(false)
```

Comment:
You call cleanup_aaa from both setup and teardown. But you only do the feature disable from teardown? If it makes sense for teardown then it should also occur for setup. You can't assume that every testbed starts out clean.

Unless you plan to use cleanup_aaa for something beyond setup & teardown then it makes sense to include the feature disable inside cleanup_aaa.